security/postmortem-template.md

```diff
@@ -0,0 +1,70 @@
+> Slimmed down template from: Betsy Beyer, Chris Jones, Jennifer Petoff, and Niall Richard
+> Murphy. [“Site Reliability
+> Engineering.”](https://landing.google.com/sre/book/chapters/postmortem.html),
+> modified from
+> https://raw.githubusercontent.com/dastergon/postmortem-templates/master/templates/postmortem-template-srebook.md.
+> Follow the SRE link for examples of how to populate.
+
+# Security postmortem for CVE-YEAR-ABCDEF
```

Comment:
Do we want to do this as a GH issue that is opened in draft state, folks comment on and the author edits? Alternatively includes a Google Doc or a PR.

Reply:
I'm fine with any solution, though I would lean towards a PR and we submit the PMs somewhere in the repo where it's easier to find them.